src/DevChatter.DevStreams.Infra.Dapper/Services/ChannelAggregateService.cs

```diff
@@ -92,6 +92,34 @@ public Channel GetAggregate(int id)
             }
         }
 
+        /// <summary>
+        /// Returns all the channel data for a channel object.
+        /// </summary>
+        /// <returns>All Channel Info in a List<Channel> Object </Channel></returns>
+        public async Task<List<Channel>> GetAllAggregate()
+        {
+            string sql = @"SELECT * FROM Channels;
+                         SELECT * FROM TwitchChannels;";
+            
+            using (IDbConnection connection = new SqlConnection(_dbSettings.DefaultConnection))
+            {
+                var dbQuery = await connection.QueryMultipleAsync(sql);
+                List<Channel> channels = dbQuery.Read<Channel>().ToList();
+                List<TwitchChannel> twitchChannels = dbQuery.Read<TwitchChannel>().ToList();
+
+                foreach (var channel in channels)
+                {
+                    if (twitchChannels.Where(x => x?.ChannelId == channel?.Id).Any())
+                    {
+                        channel.Twitch = twitchChannels.Where(x => x?.ChannelId == channel?.Id).First();
```

Comment:
Get rid of the outer `if` and make the `Where` into a `SingleOrDefault` that includes the filtering lambda expression.